src/main/java/org/folio/rest/impl/LoginAPI.java

```diff
@@ -307,27 +300,16 @@ private Future<String> handleLogin(LoginCredentials credentials, String tenantId
             .map(obj -> ((JsonObject)obj).mapTo(UserTenant.class))
             .collect(toList());
           if (userTenants.isEmpty()) {
-            // No matching tenant - if a tenant was specified, return an error
-            if (StringUtils.isNotBlank(credentials.getTenant())) {
-              logger.info(MISSING_USER_TENANT_ASSOCIATION_LOG, credentials.getUsername(), credentials.getUserId(),
-                  credentials.getTenant());
-              asyncResultHandler.handle(Future.succeededFuture(PostAuthnLoginResponse.respond422WithApplicationJson(
-                  LoginAPI.getErrors(MISSING_USER_TENANT_ASSOCIATION, CODE_MISSING_USER_TENANT_ASSOCIATION))));
-              promise.fail(new Exception(MISSING_USER_TENANT_ASSOCIATION));
-            } else {
-              promise.complete(tenantId);
-            }
+            // No matching tenant - if a tenant was specified
+            promise.complete(tenantId);
           } else if (userTenants.size() > 1) {
-            // Multiple matching user-tenants, return an error with the list
+            // Multiple matching user-tenants
```

Comment:
please adjust this comment to: Multiple matching user-tenants, using tenant from okapi headers

Reply:
updated comment.